source/leuven/genius_2_rocky.rst

```diff
@@ -1,121 +1,267 @@
+stack are available, is by using the cluster module. It is however also
+possible to manually modify the path where modules are searched.
 
+Each software stack is located in a directory with the following hierarchical
+structure::
 
+   /apps/leuven/${VSC_OS_LOCAL}/${VSC_ARCH_LOCAL}${VSC_ARCH_SUFFIX}/TOOLCHAIN_VERSION/modules/all
 
-Using cluster modules
-~~~~~~~~~~~~~~~~~~~~~
+e.g.:
 
-Instead of working with a single default, unchanging, modulepath, refering to a single default toolchain, a new approach is taken. Each cluster partition will have a so called cluster module that sets the ``$MODULEPATH`` that is valid for the specific nodes in the cluster partition. The cluster module will detect the underlying CPU architecture and uses this for setting the path correctly.
+.. code-block:: shell
 
+   /apps/leuven/rocky8/skylake/2018a/modules/all
 
-.. _check_available_software:
+This convention is in line with other VSC sites and will also be used on wICE
+and future clusters. In order to add such a directory to your modulepath, the
+following command can be used:
 
-Check available software
-~~~~~~~~~~~~~~~~~~~~~~~~
-On the login node you will be able to load the different available cluster modules (only for experimentation, not for actual computing). In order to do this you can query the available cluster moduels with::
+.. code-block:: shell
 
-   $ module avail
+   module use /apps/leuven/rocky8/skylake/2018a/modules/all
 
-this wil show the available cluster modules::
+To remove the entry again:
 
-   --------------------------- /apps/leuven/etc/modules ---------------------------
-   cluster/default
-   cluster/genius/amd_long               (S)
-   cluster/genius/amd                    (S)
-   cluster/genius/batch_debug            (S)
-   cluster/genius/batch_long             (S)
-   cluster/genius/batch                  (S)
-   ...
-   cluster/wice/batch
-   ...
+.. code-block:: shell
 
-Loading any of this modules on the login node::
+   module unuse /apps/leuven/rocky8/skylake/2018a/modules/all
 
-   $ module load cluster/genius/batch
+.. _impact_on_user_installed_software:
 
-will set the module path for the modules that are applicable for the Genius ``batch`` partition::
+Impact on user-installed software
+---------------------------------
+If you have installed a software package yourself in your own account, and you
+did this on a Genius CentOS 7 node, it must be recompiled on Genius on a node
+with the new OS. This can be done on one of the available test nodes. Please
+request access to the lpt2_rocky8_pilot group.
 
-   $ module load cluster/wice/batch
+Conda environments
+----------------------------
+The Conda environment you installed might need reinstallations. If you already
+have a Conda environment that works on wICE, it also should work on Genius
+after the migration. If you only have a Conda environment working on Genius,
+it is best to create a new Conda installation. In this case, it is recommended
```

Comment:
An additional remark can also be given regarding `pip` installs. Something along this line:

"If you used ``pip`` to install a software inside your conda environment and ``pip`` has compiled the package from source, you certainly need to recreate your conda environment."